docs/blog/newsletter-june-2020.rst

```diff
@@ -0,0 +1,132 @@
+.. post:: June 03, 2020
+   :tags: newsletter
+
+#######################################
+Write the Docs Newsletter – June 2020
+#######################################
+
+Hey all - it’s Beth and the newsletter team here.
+
+It feels like more and more often that I write to you while the backdrop is bleak, and it would be impossible tonot talk about what’s happening in the US. `Inclusion is at the core of Write the Docs </about-us/#inclusivity>`__; we are distraught by the suffering, oppression and systemic racism the Black community faces every day. This is much better said in our `statement of support </blog/we-stand-with-the-black-community/>`__, but in particular I wanted to mention that we are matching donations, up to a total of $1,500, to non-profits working to support the Black community. See the `blog post </blog/we-stand-with-the-black-community/>`__ for how to tell us about your donation so that we can match it.
```

Comment:
I think "in the US" is probably not right here, but I struggled really hard with how to refer to whats happening. Maybe:

```suggestion
It feels like more and more often that I write to you while the backdrop is bleak, and it would be impossible to not talk about the ongoing protests around the world. `Inclusion is at the core of Write the Docs </about-us/#inclusivity>`__; we are distraught by the suffering, oppression and systemic racism the Black community faces every day. This is much better said in our `statement of support </blog/we-stand-with-the-black-community/>`__, but in particular I wanted to mention that we are matching donations, up to a total of $1,500, to non-profits working to support the Black community. See the `blog post </blog/we-stand-with-the-black-community/>`__ for how to tell us about your donation so that we can match it.
```